chapters/planning-data-work.tex

```diff
@@ -151,162 +469,185 @@ \subsection{Code management}
 but if the code is well-organized, they will be much easier to make.
 Below we discuss a few crucial steps to code organization.
 They all come from the principle that code is an output by itself,
-not just a means to an end.
-So code should be written thinking of how easy it will be for someone to read it later.
-
-Code documentation is one of the main factors that contribute to readability,
-if not the main one.
-There are two types of comments that should be included in code.
-The first one describes what is being done.
-This should be easy to understand from the code itself if you know the language well enough and the code is clear.
-But writing plain English (or whichever language you communicate with your team on)
-will make it easier for everyone to read.
-The second type of comment is what differentiates commented code from well-commented code:
-it explains why the code is performing a task in a particular way.
-As you are writing code, you are making a series of decisions that
-(hopefully) make perfect sense to you at the time.
-However, you will probably not remember how they were made in a couple of weeks.
-So write them down in your code.
-There are other ways to document decisions
-(GitHub offers a lot of different documentation options, for example),
-but information that is relevant to understand the code should always be written in the code itself.
+not just a means to an end,
+and should be written thinking of how easy it will be for someone to read it later.
 
-Code organization is the next level.
-Start by adding a code header.
-This should include simple things such as stating the purpose of the script and the name of the person who wrote it.
+Code documentation is one of the main factors that contribute to readability.
+Start by adding a code header to every file.
```

Comment:
I think we need to explain what a _code header_ is as that is not obvious if you haven't seen one. Have we introduced _comments_ by this point?

Reply:
986930ce31bea45f857c944877430b2232aae69e